README.md

```diff
@@ -10,7 +10,7 @@ of your source files for more accurate results.
 [![Build Status](https://dev.azure.com/jpsim/SwiftLint/_apis/build/status/realm.SwiftLint?branchName=main)](https://dev.azure.com/jpsim/SwiftLint/_build/latest?definitionId=4?branchName=main)
 [![codecov.io](https://codecov.io/github/realm/SwiftLint/coverage.svg?branch=main)](https://codecov.io/github/realm/SwiftLint?branch=main)
 
-![](assets/screenshot.png)
+![](https://github.com/realm/SwiftLint/blob/main/assets/screenshot.png?raw=true)
```

Comment:
Instead of `?raw=true` you can use `https://raw.githubusercontent.com/realm/SwiftLint/main/assets/screenshot.png`.

Reply:
Don't forget to prepend `https://raw.githubusercontent.com` as well. 😉